### ISSUE TITLE
Can't sign Ethereum transaction offline on v0.11.2

### ISSUE THREAD

**Issue by fountain-github** [trezor/python-trezor#371]
Here is the traceback:
```
Traceback (most recent call last):
  File "/usr/local/bin/trezorctl", line 1892, in <module>
    cli()  # pylint: disable=E1120
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/usr/local/bin/trezorctl", line 1266, in ethereum_sign_tx
    address = "0x%s" % ethereum.get_address(client, address_n).hex()
AttributeError: 'str' object has no attribute 'hex'
```

**Comment by matejcik** [comment]
this is fixed in master via #366 and #365 

i'll bump the version so that you can install directly from github. not sure if we want to do a release just for this, but maybe yes

**Comment by matejcik** [comment]
version bumped.
not doing a pypi release for now, but it's possible to set a version requirement to 0.11.3
